doc/src/challenges/0014-convert-num.md

```diff
@@ -0,0 +1,140 @@
+# Challenge 14: Safety of Primitive Conversions
+
+- **Status:** Open
+- **Tracking Issue:** https://github.com/model-checking/verify-rust-std/issues/220
+- **Start date:** 2024/12/15
+- **End date:** 2025/2/28
+- **Prize:** *TBD*
+
+-------------------
+
+## Goal
+
+Verify the safety of number conversions in `core::convert::num`.
+
+There are three classes of conversions that use unsafe code. All conversions use macros to implement traits for primitive types in bulk. You will need to add contracts into the macros so that contracts are generated for each implementation. As the conversions are all macro generated, it is probably a good idea to write your own macro for generating the harnesses.
+
+### Success Criteria
+
+#### Safety for Float to Int
```

Comment:
It would be nice to also list all of the types that implement this float to int functionality, similarly to how you have the list in the section below for conversions, so that we're explicit about which proofs we expect.

Reply:
I moved the full list of expected contracts to the Success Criteria section. I also added a note about having a panicking harness for the `try_from` contracts.

Reply:
Thanks -- I suggested putting the NonZero header back, since I think it was better to divide between the NonZero work and the float to int work. I meant just adding the implementations for float to int like you had for NonZero--added a suggestion below to that effect.

Reply:
Thanks, sorry about the misunderstanding.

Reply:
No worries!